src/grimoirelab/core/config/settings.py

```diff
@@ -285,11 +283,12 @@
 }
 
 RQ_QUEUES = {
-    Q_PERCEVAL_JOBS: _RQ_DATABASE,
-    Q_STORAGE_ITEMS: _RQ_DATABASE,
-    Q_EVENTS: _RQ_DATABASE,
+    Q_EVENTIZER_JOBS: _RQ_DATABASE,
 }
 
+EVENTS_STREAM_NAME = os.environ.get('GRIMOIRELAB_EVENTS_STREAM_NAME', 'events')
+EVENTS_STREAM_MAX_LENGTH = int(os.environ.get('GRIMOIRELAB_EVENTS_STREAM_MAX_LENGTH', 2 * 10 ** 6))
```

Comment:
This looks like a magic number. What is it and why is that the length?

Reply:
This is the stream's limit. Once the stream reaches this capacity, new items will replace older ones. On average, each Git event consumes approximately 2.6 Kb (calculated using MEMORY USAGE). While this size can be reduced, it is important for consumers to read from the stream before it reaches its limit; otherwise, items will be discarded.

I can lower this number, but it depends on the memory of the Redis server.